src/chrome/content/rules/EFF.xml

```diff
@@ -1,17 +1,26 @@
-<!--
-	Other EFF rulesets:
-		- Freerangekitten.com.xml
-		- IFightSurveillance.org.xml
-		- ManilaPrinciples.org.xml
-		- Panopticlick.com.xml
```

Comment:
The targets added in this PR are already covered in these files. We shouldn't merge them back into one file.